cogs/quotes.py

```diff
@@ -22,12 +24,50 @@ async def quote(self, ctx, search=""):
     @quote.command()
     @commands.has_permissions(administrator=True)
     async def add(self, ctx, *, content):
+        """adds a quote"""
+
         content = await commands.clean_content(fix_channel_mentions=True).convert(ctx, content)
 
         with self.bot.db.get(ctx.guild.id) as db:
-            db.execute("INSERT INTO quotes (content) VALUES (?)", (content, ))
+            db.execute("INSERT INTO quotes (content) VALUES (?)", (content,))
 
         await ctx.message.add_reaction('\U00002705')
 
+    @quote.command()
+    @commands.has_permissions(manage_channels=True)
+    async def list(self, ctx, search=""):
+        """Lists all quotes that use an optional query"""
+
+        # if else for performance reasons, as SQLite does not perform query optimisation
```

Comment:
This is a Discord bot, written in Python. We're not at a point where we need to care about SQLite query optimization.

Reply:
The point is that with this command all the quotes can be pulled down (only by the ones with elevated Priviliges), but anyway..

since @Kroko-fant stated, that he was displeased with the resource utilisation of the sql server I thought this was a good Idea..

but yea.. you are right..